autorel/changelog_generator/github_plugin.py

```diff
@@ -0,0 +1,101 @@
+## Helper functions to fetch Pull Requests & Issues information from github
+## To support GitLab and other platforms, similar helper functions should be wrote
+from github import Github
+from settings import (AUTH_TOKEN,
+                      PROJECT
+                      )
+from structures import (PullRequest,
+                        Issue,
+                        Contributor,
+                        Commit
+                        )
+
+
+# Instantiate the client
+cli = Github(AUTH_TOKEN)
+
+
+def getPull(pull_id):
```

Comment:
Is there any reason why these functions are not wrapped in a class? We could use dependency injection in `ChangelogGenerator` in that case (with an additional interface).